old/production-configs/jupyter.openfisca.fr/config/htpasswd

```diff
@@ -0,0 +1,2 @@
+# Generated with "openssl passwd"
+OpenFisca:5GRDL4T9EdJds
```

Comment:
We should never commit a password on a public git repository, even for outdated services. 
We should make sure this doesn't create a security risk. Totally deactivate `jupyter.openfisca.fr` might be enough. Where is this file coming from?

Reply:
😞  This file was in the `production-configs` directory. 
I deleted the nginx config for `jupyter.openfisca.fr`

Reply:
I'm not seeing the file in the git history, so `git filter-branch` won't do anything more. 

I think this file was only used in the `jupyter.openfisca.fr` conf, so if this app is shut down, the risk is probably low.